### DISCUSSION TITLE
possibility to use g-helper only for mouse binds and parameters editing ?

### DISCUSSION THREAD

**Discussion by nassimooo** [seerge/g-helper#1512] [Q&A]
Hello there,

After experiencing hell on earth with Armoury Crate i was searching for an alternative software for customizing my M4 wireless mouse and i've found this software, but if i understand it correctly it can only be used on ASUS laptop (tried to install ASUS System Control Interface anyway since i have an ASUS motherboard but i couldn't) i was wondering if there's a possibility to install it on any computer just to use the mouse control panel of g-helper or am i doomed to use my mouse without any customization or battery level check ?

If it's impossible is there another software for controlling and editing my mouse ?

Thanks in advance and sorry if the question is redundant.

**Comment by seerge** [comment]
@nassimooo currently 95% of the app is laptop specific :) so it won't launch on desktops or non asus laptops at all :)

**Comment by IceStormNG** [comment]
> If it's impossible is there another software for controlling and editing my mouse ?

Not yet. But someone could take the part for the mouse and make a separate app with just the mouse control part. I made the mouse code as independent as possible.
Though, I don't have the time to do that, and I guess @seerge is busy with G-Helper on its own already.

**Comment by nassimooo** [comment]
Thanks for the answers ! Hopefully a tiny version of g-helper will rise one day with just the mouse related stuff !

**Comment by spd789562** [comment]
Is there any other way to disable dpi or fix the mouse dpi so nothing happens when misclicking dpi adjust button? I just don't want install Armoury Crate just for that function

**Reply by IceStormNG** [reply]
What is the issue? If the mouse DPI for your mouse does not change as it should, then please open an issue for that specific mouse because it could be that the mouse profile is broken.
We will gonna fix it then.

**Reply by spd789562** [reply]
wait no, it not  your program's problem, just think there has a tool to fixed the mouse DPI so that DPI doesn't change when hit DPI adjust button on the mouse.
but seems this tool not showing my ROG Gladious III in the panel cause my PC is desktop not laptop
